### PR TITLE
Update Binder, standalone installer caveats & Big Sur in Download section

### LINKED ISSUE #209: Update binder link
The link for starting Spyder on binder does not work. There is some explanation in https://github.com/spyder-ide/spyder/issues/19832#issuecomment-1287602906 and other comments on that issue.

### PR DESCRIPTION
**Pull request by CAM-Gerlach**
Part of spyder-ide/spyder-docs#334

* Update the outdated, broken Binder link
* Revise the download section to reflect the limitations of the standalone installers
* Refine the language in that section and other small related updates
* Elide the mostly out of date (AFAIK) information about macOS Big Sur

Fixes #209

### CODE REVIEW

File: content/contents.lr [hunk 1]
```diff
@@ -214,16 +214,13 @@ full_wdith: false
 content:
 
 Want to join the community of scientists, engineers and analysts all around the world using Spyder?
-Click the button below to download the suggested installer for your platform; we offer standalone installers on Windows and macOS.
-For Linux, we recommend the cross-platform Anaconda distribution, which includes Spyder and many other useful packages for scientific Python.
-You can also try out Spyder right in your web browser by [launching it on Binder](https://mybinder.org/v2/gh/spyder-ide/binder-environments/spyder-stable?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252Fjuanis2112%252FSpyder-Workshop%26urlpath%3Ddesktop%252F%26branch%3DSolutions).
+Click the button below to download the suggested installer for your platform.
+We offer standalone installers on Windows and macOS, and as our Linux installer is are still experimental, we currently recommend the cross-platform Anaconda distribution for that platform, which includes Spyder and many other useful packages for scientific Python.
```
Comment (ccordoba12) [inline content/contents.lr]:
```suggestion
We offer standalone installers on Windows and macOS, and as our Linux installer is are still experimental, we currently recommend the cross-platform Anaconda distribution for that operating system, which includes Spyder and many other useful packages for scientific Python.
```

Avoids repeating `platform` twice in this sentence.

File: content/contents.lr [hunk 2]
```diff
@@ -214,16 +214,13 @@ full_wdith: false
...
+You can also try out Spyder right in your web browser by [launching it on Binder](https://mybinder.org/v2/gh/spyder-ide/binder-environments/spyder-stable?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252Fspyder-ide%252FSpyder-Workshop%26urlpath%3Ddesktop%252F%26branch%3Dmaster).
 
-For a detailed guide on the many different methods of obtaining Spyder, please refer to our full [installation instructions](https://docs.spyder-ide.org/current/installation.html), and check out our [release page](https://github.com/spyder-ide/spyder/releases/latest) for links to all our installers.
-These approaches are generally intended for experienced users and those with specific needs, so we recommend sticking with the recommended installer unless you have a specific reason to go with another.
+Standalone installer support for third-party Spyder plugins and package/environment management is still under development, so we advise installing a Conda-based Python distribution for these use cases.
```
Comment (ccordoba12) [inline content/contents.lr]:
```suggestion
Support for installing additional Python packages or third-party Spyder plugins and environment management is still under development, so we advise installing a Conda-based Python distribution for these use cases.
```

Make this sentence a bit easier to read.

Comment (CAM-Gerlach) [reply content/contents.lr]: I can see the concern with the original, but I at least find the suggested syntax to actually be a good deal _more_ confusing, complex and hard to parse. Here's a start at splitting this into two sentences

```suggestion
The standalone installers don't yet work with third-party plugins, so users needing that should use Spyder through a Conda-based distribution instead.
Also, they don't currently support installing custom Python packages beyond the basic set pre-installed, so most users will want to have an external Python environment to run their own code, like with any other IDE.
```

File: content/contents.lr [hunk 3]
```diff
@@ -214,16 +214,14 @@ full_wdith: false
...
+You can also try out Spyder right in your web browser by [launching it on Binder](https://mybinder.org/v2/gh/spyder-ide/binder-environments/spyder-stable?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252Fspyder-ide%252FSpyder-Workshop%26urlpath%3Ddesktop%252F%26branch%3Dmaster).
 
-For a detailed guide on the many different methods of obtaining Spyder, please refer to our full [installation instructions](https://docs.spyder-ide.org/current/installation.html), and check out our [release page](https://github.com/spyder-ide/spyder/releases/latest) for links to all our installers.
-These approaches are generally intended for experienced users and those with specific needs, so we recommend sticking with the recommended installer unless you have a specific reason to go with another.
+The standalone installers don't yet work with third-party plugins, so users needing that should use Spyder through a Conda-based distribution instead.
+Also, they don't currently support installing custom Python packages beyond the basic set pre-installed, so most users will want to have an external Python environment to run their own code, like with any other IDE.
```
Comment (ccordoba12) [inline content/contents.lr]: I still think that mentioning that our installers can't handle additional Python packages should be said first because it's important. But I'm not going to push too hard on that.

So, if you feel this is fine, then I'm ok with it too.

Comment (CAM-Gerlach) [reply content/contents.lr]:
```suggestion
The built-in interpreter of the standalone version doesn't currently support installing custom packages beyond the common scientific libraries bundled with it, so most users will want to have an external Python environment to run their own code, like with any other IDE.
Also, the standalone installers don't yet work with third-party plugins, so users needing them should use Spyder through a Conda-based distribution instead.
```

I can switch these here; I don't have a strong preference. I only had it the way it was because it was easier to word it that way and avoid doubling up "install", but it doesn't matter _that_ much. I've also tweaked some other things.

File: content/contents.lr [hunk 4]
```diff
@@ -214,16 +214,14 @@ full_wdith: false
...
+You can also try out Spyder right in your web browser by [launching it on Binder](https://mybinder.org/v2/gh/spyder-ide/binder-environments/spyder-stable?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252Fspyder-ide%252FSpyder-Workshop%26urlpath%3Ddesktop%252F%26branch%3Dmaster).
 
-For a detailed guide on the many different methods of obtaining Spyder, please refer to our full [installation instructions](https://docs.spyder-ide.org/current/installation.html), and check out our [release page](https://github.com/spyder-ide/spyder/releases/latest) for links to all our installers.
-These approaches are generally intended for experienced users and those with specific needs, so we recommend sticking with the recommended installer unless you have a specific reason to go with another.
+The built-in interpreter of the standalone version doesn't currently support installing custom packages beyond the common scientific libraries bundled with it, so most users will want to have an external Python environment to run their own code, like with any other IDE.
```
Comment (ccordoba12) [inline content/contents.lr]:
```suggestion
The built-in interpreter of the standalone version doesn't currently support installing or using other packages beyond the common scientific libraries bundled with it, so most users will want to have an external Python environment to run their own code, like with any other IDE.
```

Small tweaks, then this should be ready for me.

Comment (CAM-Gerlach) [reply content/contents.lr]: What about just eliding "custom" and leaving it at that? To the extent "using" is contingent on "installing" first, "using" is redundant and it wouldn't be an either/or relationship as the statement implies, and to the extent that its not, the statement isn't accurate since users can still use any package they want without installing just as much they can with a conda install, by placing it in their working directory.

```suggestion
The built-in interpreter of the standalone version doesn't currently support installing packages beyond the common scientific libraries bundled with it, so most users will want to have an external Python environment to run their own code, like with any other IDE.
```

Comment (ccordoba12) [reply content/contents.lr]: Ok, sounds good to me.

**Review by ccordoba12 — APPROVED** [review]
Looks good to me now, thanks @CAM-Gerlach!

**Comment by ccordoba12** [comment]
Merging because I think the improvements here are beneficial for users. If there are additional comments about this, they can be solved in follow-up PRs.
